package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
+    "react": "^17.0.2",
```

Comment:
- [ ] Kindly make sure `react-redux` is present in `package.json` by running the `npm install react-redux` command on your terminal. Because the `react-redux` will be needed to use Redux in the next projects